### DISCUSSION TITLE
get_plugin api

### DISCUSSION THREAD

**Discussion by kranskydog** [jhuckaby/Cronicle#664] [Ideas]
if I use the API to create an event, I need to know what are the parameters the plugin expects. While I can get a list of all the plugins and their parameters, I can't just get the specific plugin I am using.

can an additional API be created to get_plugin instead of having to do get_plugins and then looking for the one I need from the response?

**Comment by jhuckaby** [comment]
Sure thing, will add to the TODO list.
